lib/active_shipping/carriers/fedex.rb

```diff
@@ -621,11 +621,12 @@ def parse_tracking_response(response, options)
 
           location = Location.new(:city => city, :state => state, :postal_code => zip_code, :country => country)
           description = event.at('EventDescription').text
+          type_code = event.at('EventType').text
 
           time          = Time.parse(event.at('Timestamp').text)
           zoneless_time = time.utc
 
-          shipment_events << ShipmentEvent.new(description, zoneless_time, location)
+          shipment_events << ShipmentEvent.new(description, zoneless_time, location, description, type_code)
```

Comment:
Yeah, it's always been using `description` for `name`, I just went ahead and passed `description` for `message` as well instead of passing in `nil`.